lib/akamod/enrich-type.py

```diff
@@ -49,10 +49,8 @@ def enrichtype(body, ctype,
     except KeyError:
         # In this case, sourceResource is not present, so give up and return
         # the original data unmodified.
```

Comment:
I'm assuming we get warned about missing SR elsewhere, so this was redundant?

Reply:
Yeah this is duplicating effort